domain-registrars/availability-checks.md

```diff
@@ -22,7 +22,7 @@ In addition to the regular [Common Module Parameters](/domain-registrars/module-
 | --------- | ----------- | ----------- |
 | searchTerm | string | The search term provided by the end user
 | punyCodeSearchTerm | string | For an IDN domain, the puny code encoded search term
-| tldsToInclude | array | An array of TLDs/extensions to perform the availability check for
+| tldsToInclude | array | An array of Spotlight TLDs/extensions to perform the availability check for
```

Comment:
tldsToInclude is used for spotlight and namespinning. Naming it as spotlight would be invalid.